rules/falco_rules.yaml

```diff
@@ -82,6 +82,9 @@
 - macro: spawned_process
   condition: (evt.type in (execve, execveat) and evt.dir=<)
 
+- macro: successful_spawned_process
+  condition: (evt.type in (execve, execveat) and evt.dir=< and evt.arg.res=0)  
```

Comment:
I think we should use `evt.rawres` or `evt.rawarg.res` because the `evt.arg` is going to resolute the return value, e.g. `ENOENT`.

Reply:
From my tests the `evt.arg.res` field is not resolved if it's equal to `0`, anyway I'll use the suggested field to be sure that this change will survive to future mappings on this field. Ty again for the precious suggestions